psga/config/psga-schema-sars-cov-2.yaml

```diff
@@ -0,0 +1,475 @@
+pipeline:
+  name: sars-cov-2
+  docker_image: ''
+  protocols:
+
+    # ARTIC_V1
+    - name: illumina-sars-cov-2-articv1
+      kit:
+        name: ARTIC_V1
+        slug: articv1
+      sequencer_technology:
+        name: illumina
+        slug: illumina
+      pathogen:
+        name: Sars-CoV-2
+        slug: sars-cov-2
```

Comment:
@cramshaw how is the `name` used? The correct name is `SARS-CoV-2`, but it isn't a pipeline parameter. Is this used by the backend?

Reply:
this is just used by the backend - when we create the pipeline it needs to be connected in the database to a pathogen